### PR TITLE
Enable a11y addon for accessibility testing

### PR DESCRIPTION
**Pull request by jjpxiii**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -143,7 +144,7 @@
         "./Pagination": "./dist/Pagination.js",
         "./Notice": "./dist/Notice.js",
         "./Modal": "./dist/Modal.js",
-        "./MainNavigation": "./dist/MainNavigation/index.js",
+        "./MainNavigation": "./dist/MainNavigation\\index.js",
```
Comment (garronej) [inline package.json:147]:
```suggestion
        "./MainNavigation": "./dist/MainNavigation/index.js",
```

Comment (jjpxiii) [reply package.json:147]: Oops I'm sorry I didn't see that, I guess you're not supposed to contribute from a Windows machine with prettier 😬

**Comment by jjpxiii** [comment]
Thanks Joseph, I'll be more careful next time and I also will try to make the build scripts working on Windows, what do you think ?

**Comment by garronej** [comment]
Thank you very much for your proposal.  
Don't worry about it though, I just fixed it, I think.  
If you are willing to contribute futher there is many more miningfull things on the todo list :)  
Anyway thank you for that!

**Pull request merged**